README.md

```diff
@@ -65,21 +65,31 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 > **NOTE:** PyTorch 2.2.1 does not support `torch.compile` with Python 3.12. On Fedora 39+, install `python3.11-devel` and create the virtual env with `python3.11` if you wish to use PyTorch's JIT compiler.
 
 ## ✅ Getting started
-
 ### 🧰 Installing `ilab`
 
-1. If you are on Fedora Linux, install C++, Python 3.9+, and other necessary tools by running the following command:
+1. You first need to install python version 3.9+ and C++.
 
-   ```shell
-   sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
-   ```
+    *Installing packages on Linux (Fedora)*
```

Comment:
Should these be markdown headings so they can be linked to?

Reply:
+1, also `Python` should be capitalized